docs/source/app-auth-update.rst

```diff
@@ -0,0 +1,43 @@
+App Integration Auth Updating
+=============================
+
+Overview
+--------
+
+In the instance that the required authentication information for a configured App changes, it may be necessary to update a currently deployed App to reflect this.
```

Comment:
Wordy, thoughts on just saying `You may need to change an App's credentials due to internal rotation policies or otherwise. The StreamAlert CLI allows you to easily update App credentials.`?